docs/manual/Makefile

```diff
@@ -71,7 +71,7 @@ manual.html: manual.pdf CFLogo.png favicon-checkerframework.png ../api
 	sed -i -e "s%<style type=\"text/css\">%<style type=\"text/css\">\nimg { max-width: 100\%; max-height: 100\%; }%" manual.html
 
 ../api:
-	cd .. && ln -s ../checker/api api
+	cd .. && ln -fs ../checker/api api
```

Comment:
Thanks for the explanation.  The commit message won't be visible to people reading the source, but anyone who chooses to change this in the future will be able to see it.